DigitalLearningSolutions.Web/Views/Supervisor/SignOffProfileAssessment.cshtml

```diff
@@ -1,207 +1,216 @@
+            @(Model.SelfAssessmentResultSummary.RoleName.Length > 35 ? Model.SelfAssessmentResultSummary.RoleName.Substring(0, 32) + "..." : Model.SelfAssessmentResultSummary.RoleName)
+          </a>
+        </li>
+        <li class="nhsuk-breadcrumb__item">
+          Sign-off Self Assessment
+        </li>
+      </ol>
+    </div>
+    <p class="nhsuk-breadcrumb__back">
+      <a class="nhsuk-breadcrumb__backlink" asp-controller="Supervisor"
+         asp-action="ReviewDelegateSelfAssessment"
+         asp-route-supervisorDelegateId="@ViewContext.RouteData.Values["supervisorDelegateId"]"
+         asp-route-candidateAssessmentId="@Model.SelfAssessmentResultSummary.ID">
+        Back to @(Model.SelfAssessmentResultSummary.RoleName.Length > 35 ?
+                  Model.SelfAssessmentResultSummary.RoleName.Substring(0, 32) + "..." : Model.SelfAssessmentResultSummary.RoleName)
+      </a>
+    </p>
+  </nav>
+}
+
+<details class="nhsuk-details nhsuk-expander">
+  <summary class="nhsuk-details__summary">
+    <h1 class="nhsuk-details__summary-text nhsuk-u-margin-bottom-0">
+      @Model.SupervisorDelegate.FirstName @Model.SupervisorDelegate.LastName
+    </h1>
+  </summary>
+  <div class="nhsuk-details__text">
+    <partial name="Shared/_StaffDetails" model="Model.SupervisorDelegate" />
+  </div>
+</details>
+<h2>@Model.SelfAssessmentResultSummary.RoleName</h2>
+<dl class="nhsuk-summary-list">
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Assessment questions in profile
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.SelfAssessmentResultSummary.CompetencyAssessmentQuestionCount
+    </dd>
+
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Assessment question responses
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.SelfAssessmentResultSummary.ResultCount
+    </dd>
+
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Responses verified
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.SelfAssessmentResultSummary.VerifiedCount
+    </dd>
+
+  </div>
+  @if (Model.CandidateAssessmentSupervisorVerificationSummaries.Any()) {
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Response verifiers
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @foreach (var candidateAssessmentSupervisorVerificationSummary in Model.CandidateAssessmentSupervisorVerificationSummaries) {
+          <span>@DisplayStringHelper.GetPotentiallyInactiveAdminName(candidateAssessmentSupervisorVerificationSummary.Forename, candidateAssessmentSupervisorVerificationSummary.Surname, candidateAssessmentSupervisorVerificationSummary.AdminActive) (@candidateAssessmentSupervisorVerificationSummary.Email) - @candidateAssessmentSupervisorVerificationSummary.VerifiedCount</span>

```

Comment:
A pretty nasty line. Worth adding the text content to the view model?

Reply:
There isn't a mapping to a ViewModel here, and I've prefered the minimal approach for these pages made by HEE.